dev-env/windows/libexec/core.ps1

```diff
@@ -43,14 +43,15 @@ function da_auto_prevent_update {
 }
 
 function da_clear_buckets {
-    Remove-Item $scoopInstallDir\apps\scoop\current\bucket\* -r -force
+    Remove-Item $scoopInstallDir\buckets\dadew -r -force -ErrorAction Ignore
```

Comment:
For some reason the directory does not always seem to exist here so I first kill it completely ignoring failures and then recreate it.